src/documentation/pages/components/label/examples/_associate.vue

```diff
@@ -0,0 +1,71 @@
+<template lang="pug">
+  <ComponentExample title="Associate label with form elements" id="associate" :tabs="exampleTabs">
+    p.-text(slot="example-description") 
+      | Label is important for accessibility of form elements.
+    div(style="max-width: 17rem;" slot="example")
```

Comment:
```suggestion
    <Wrapper slot="example">
```

Reply:
it would require an inner <div> and make the wrapper redundant :(